plugins/mcreator-localization/lang/texts.properties

```diff
@@ -1554,6 +1555,8 @@ modelement.villagerprofession.description=Villager profession defines a version
 modelement.villagertrade.description=Villager trade adds new trades to villager professions.
 preferences.section.ui=User interface
 preferences.section.ui.description=On this page you can change user interface settings and parameters. You will need to restart MCreator if you change these settings in order for them to take effect.
+preferences.section.behaviour=Behaviour
```

Comment:
MCreator tries to use US EN across the app, not UK EN, so it should be called Behavior, the same as we call Color Color, not Colour

Reply:
Oh, we also use this form in Canada, sl it was the natural word that came to me (US always wanting to be different from others 😅)

Reply:
I was looking at the texts file and noticed I wasn't alone to write "behaviour" 
![image](https://github.com/MCreator/MCreator/assets/38427877/ac6d2045-90e1-448b-a651-bc9f97599c10)
![image](https://github.com/MCreator/MCreator/assets/38427877/3701796f-9aa3-49a3-bb3d-23387d9fcf71)
![image](https://github.com/MCreator/MCreator/assets/38427877/b4fe2c34-fff6-4767-8139-416818fb72ab)

In fact, both ways are used in the code and UI.

Reply:
Past mistakes are not reason to repeat them if we are aware of them and we found another one

Reply:
But good catch, this should be streamlined in texts.properties

Reply:
> Past mistakes are not reason to repeat them if we are aware of them and we found another one

This was only a report

Reply:
Yep, followed up on that in the second comment :)

> But good catch